recipes/polap-fmlrc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "polap-fmlrc" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "4463d4a71a8a3273108a0ab2dff0a4c37a565c05af6ee17ee83dc62404294caf" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/goshng/polap/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('polap-fmlrc', max_pin="x.x") }}
+
+requirements:
+  run:
+    - python =2.7
```

Comment:
Python2? That does not look good :-(

Reply:
I'm not sure we should add new recipes with such old dependencies. What is the use case here?

Reply:
Thank you for the reply. I am using a short-read polishing tool called FMLRC version 1 and its a package called kbchoi::msbwt, which is not part of bioconda and needs python 2.7.

Reply:
I know that this is somewhat abusing bioconda. I could not add kbchoi::msbwt package in the recipe. Instead, users could install polap-fmlrc and install kbchoi::msbwt separately.

Reply:
Sorry, we can only include packages from conda-forge or bioconda

Reply:
Thank you